docker/wsgi.py

```diff
@@ -88,6 +88,12 @@ def read_models_from_file():
 models = read_models_from_file()
 # Parse the arguments
 arguments = ArgParser.extract_args(args)
+
+# Start Redis if batching is enabled
+if arguments.batching == 'true':
+    assert arguments.redis_conf
+    subprocess.call(["redis-server", arguments.redis_conf])
```

Comment:
What happens if this fails?

Reply:
done

Reply:
Can we move this outside of wsgi, probably to setup? This is called once per worker right?